### ISSUE TITLE
Login page not displayed, prompted: Open in Authenticator?

### ISSUE THREAD

**Issue by cuzzlor** [recognizegroup/capacitor-plugin-msauth#13]
We are using Azure B2C endpoints.

Auth is working fine for Android and iOS and Web, except for when we run our app on iOS devices _with Microsoft Authenticator installed_.

When we invoke `login`, the auth window opens, but instead of showing the B2C login page, we are prompted: _Open in "Authenticator"?_

Clicking _Cancel_ results in the auth window closing and a failed login. Clicking _Open_ does the same.

Running on an iOS simulator without Microsoft Authenticator installed works fine.

Any idea why the login page URL seems to be associated with Microsoft Authenticator (and how to prevent the prompt)?

I can't find any info on why this is happening / what to do. I cannot log in to the Azure B2C tenant with Microsoft Authenticator, as neither the work nor personal sign-in pages support a B2C tenant login.

BTW thanks for your great work on this plugin, it's great to have the msal libs made available to capacitor, so the rolling session / silent login works without expiring in 60 mins and having to log in again.

![image](https://user-images.githubusercontent.com/5326367/211457140-b4e39cfc-2a1e-420a-bc58-05047b62071b.png)
![image](https://user-images.githubusercontent.com/5326367/211457249-24cec3b5-571d-4027-bb79-0f19e2a57020.png)

**Comment by bartwesselink** [comment]
Hi @cuzzlor, just to check: is the following code in the Info.plist of your XCode-project?
```
<key>CFBundleURLTypes</key>
<array>
    <dict>
        <key>CFBundleURLSchemes</key>
        <array>
            <string>msauth.$(PRODUCT_BUNDLE_IDENTIFIER)</string>
        </array>
    </dict>
</array>
<key>LSApplicationQueriesSchemes</key>
<array>
    <string>msauthv2</string>
    <string>msauthv3</string>
</array>
```

**Comment by cuzzlor** [comment]
Yes it is.

I assume it's correct since auth works as expected when running the app via iOS simulator.

**Comment by bartwesselink** [comment]
Alright. Can you create a minimal app that reproduces what you experience and share that repository with us? We have apps that use B2C using this plugin as well, which work on iOS, with the authenticator app installed.

**Comment by staff0rd** [comment]
https://github.com/recognizegroup/capacitor-plugin-msauth/issues/13

**Comment by cuzzlor** [comment]
Hi Bart,

Thanks for the nudge towards the `Info.plist` config.

This morning (on the other side of the world) we confirmed that in some situations we produced invalid `CFBundleURLSchemes` values for the B2C App Registration Reply URI.

Due to us using 2 different B2C App Registrations for local development vs deployed build (each with different `PRODUCT_BUNDLE_IDENTIFIER`), this issue _was_ caused by the bundle identifier not matching the Reply URI defined on the App Registration.

Apologies for wasting your time on a config issue. I hope that at least this issue may point someone else in the right direction if they experience the same thing.

Thanks again for your excellent work with this plugin.
